source/includes/available-permissions.rst

```diff
@@ -1,80 +1,326 @@
+        - :base_url:`Update access token <_static/api/studio.html#tag/security/operation/updateAccessToken>`
+        - :base_url:`Delete access token <_static/api/studio.html#tag/security/operation/deleteAccessToken>`
+    * - read_users
+      - System
+      - User is permitted to read users. Used in the :ref:`Users Management Console <users-management>` |br|
+        Required when using the following APIs:
+
+        - :base_url:`Get users <_static/api/studio.html#tag/users/operation/getUsers>`
+        - :base_url:`Get user <_static/api/studio.html#tag/users/operation/getUser>`
+        - :base_url:`Get user sites <_static/api/studio.html#tag/users/operation/getUserSites>`
+        - :base_url:`Get user site roles <_static/api/studio.html#tag/users/operation/getUserSiteRoles>`
+    * - update_users
+      - System
+      - User is permitted to update users. Used in the :ref:`Users Management Console <users-management>` |br|
+        Required when using the following APIs:
+
+        - :base_url:`Update user <_static/api/studio.html#tag/users/operation/updateUser>`
+        - :base_url:`Enable user <_static/api/studio.html#tag/users/operation/enableUser>`
+        - :base_url:`Disable user <_static/api/studio.html#tag/users/operation/disableUser>`
+        - :base_url:`Reset user password <_static/api/studio.html#tag/users/operation/resetUserPassword>`
+    * - rebuild_database
+      - Project
+      - User is permitted to rebuild Crafter Studio’s database and object state with the underlying repository
+    * - remove_plugins
+      - Project
+      - User is permitted to :ref:`remove installed plugins <plugin-management>` |br|
+        Required when using the following APIs:
+
+        - :base_url:`Remove plugin <_static/api/studio.html#tag/marketplace/operation/removePlugin>`
+        - :base_url:`Plugin usage <_static/api/studio.html#tag/marketplace/operation/pluginUsage>`
+    * - search_plugins
+      - Project
+      - User is permitted to :ref:`search for plugins <plugin-management>`
+    * - set_item_states
+      - Project
+      - User is permitted to set item states |br|
+        Required when using the following APIs:
+
+        - :base_url:`Set item states <_static/api/studio.html#tag/workflow/operation/setItemStates>`
+        - :base_url:`Update item states <_static/api/studio.html#tag/workflow/operation/updateItemStates>`
+    * - site_status
+      - Project
+      - User is permitted to :ref:`get status of repository for a project <project-tools-git>` |br|
+        Required when using the API :base_url:`Repository status <_static/api/studio.html#tag/repository/operation/repositoryStatus>`
+    * - unlock_repository
+      - Project
+      - User is permitted to unlock the repository
+    * - update_cluster |enterpriseOnly|
+      - System
+      - User is permitted to update cluster
+    * - read_cluster |enterpriseOnly|
+      - System
+      - User is permitted to read cluster |br|
```

Comment:
This allows the user to list the cluster members and the cluster mode (PRIMARY vs REPLICA)